run_service.sh

```diff
@@ -237,6 +273,7 @@ cd $directory
 if [ "$(git rev-parse --is-inside-work-tree)" = true ]
 then
     poetry install
+    poetry run autonomy packages sync
```

Comment:
This is required to execute the functionality in the script to swap Safe owner (it uses Open Autonomy packages).

Reply:
ok